### ISSUE TITLE
calibre upgrade doesn't work

### ISSUE THREAD

**Issue by Raffaello** [microsoft/winget-cli#1661]
### Brief description of your issue

winget upgrade calibre downloads the newer version and trigger the msi install. but nothing happen.

### Steps to reproduce

install calibre 5.23
and upgrade to 5.31 with

winget upgrade calibre

### Expected behavior

should upgrade and running the msi installer program

### Actual behavior

```
Successfully verified installer hash
Starting package install...
Successfully installed
```

but the installer wasn't triggered and required to do manually

### Environment

```shell
Windows Package Manager v1.1.12701
Copyright (c) Microsoft Corporation. All rights reserved.

Windows: Windows.Desktop v10.0.19043.1288
Package: Microsoft.DesktopAppInstaller v1.16.12701.0
```
```

**Comment by Raffaello** [comment]
actually i have just notice it could be a "conflict" among 32 and 64 bits versions. as installed the 64 bits but the older version was 32 bit. And this lead to winget not understanding clearly that the 32bit version has been upgrade to the modern version with 64. 
Also now there are 2 installed version.

**Comment by jedieaston** [comment]
I think Calibre 32 bit and 64 bit are designed to be able to be installed at the same time, if you installed 32-bit Calibre winget doesn't check the architecture of the installed version, instead assuming that you made the correct choice the first time and it can just install the architecture for your system.

**Comment by Raffaello** [comment]
yes both can be installed at the same time. in my case i installed 32bit and winget upgraded installing 64bit, but still detecting 32bit. anyway.. cleaned up the installation and replaced with 64 bit and all good.
